deereflow/deereflow-Fairfax.md

```diff
@@ -0,0 +1,5 @@
+## Fairfax VA :heart:
+
+#### Restaurants
```

Comment:
@deereflow I love ribs and steaks :meat_on_bone: Do you have a favorite steak place?